SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,19 @@
-| `schema` | Contains the schema files required for the GDK to interact with SpatialOS.
-| `Scripts/` | Contains a set of helper scripts used to build a GDK worker.
-| `Source/Programs/` | Contains the Unreal SDK code generator.
-| `Source/SpatialGDK/Legacy/` | Contains the source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS).
-| `Source/SpatialGDK/Legacy/Deprecated/` | Contains the forked source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS) that the GDK does not depend on. This is present purely for backwards compatibility.
-| `Source/SpatialGDK/Public/WorkerSdk/` | (Not tracked in git) Contains the [C++ worker SDK](https://docs.improbable.io/reference/latest/cppsdk/introduction) headers. You install these when you run `BuildGDK.bat`
-| `Source/SpatialGDK/Public` | Public source code of the GDK for Unreal module.
-| `Source/SpatialGDK/Private` | Private source code of the GDK for Unreal module.
+| `SpatialGDK/Binaries/ThirdParty/Improbable/` | (Not tracked in git) This directory contains all binaries required for your game project with the GDK for Unreal. These files are generated by `BuildGDK.bat`.
+| `SpatialGDK/Build/core_sdk/` | (Not tracked in git) Contains the [C API worker SDK](https://docs.improbable.io/reference/latest/capi/introduction) depndencies used by the Unreal GDK to serialize data to and from SpatialOS.
+| `SpatialGDK/Build/Programs/` | Contains the source and project files for the executables used when building the GDK.
+| `SpatialGDK/Build/Scripts/` | Contains a the helper scripts that allow you to build either a server or a client worker.
+| `SpatialGDK/Documentation` | Contains the documentation for the GDK for Unreal.
+| `SpatialGDK/Extras/fastbuild` | Contains files related to [FASTBuild](http://www.fastbuild.org/docs/home.html), an open-source build system that is currently only usable by Improbable engineers.
+| `SpatialGDK/Extras/schema` | Contains the [schema files](https://docs.improbable.io/reference/latest/shared/glossary#schema) required for the GDK to interact with SpatialOS.
+| `SpatialGDK/Extras/linting/` | Contains the scripts we use to lint the Unreal GDK.
+| `SpatialGDK/Source/SpatialGDK/Public/WorkerSdk/` | (Not tracked in git) Contains the [C API worker SDK](https://docs.improbable.io/reference/latest/capi/introduction) headers which are used while building the GDK. You install these when you run `Setup.bat`
+| `SpatialGDK/Source/SpatialGDK/Public` | Contains the public source code of the GDK for Unreal module.
+| `SpatialGDK/Source/SpatialGDK/Private` | Contains the private source code of the GDK for Unreal module.
```

Comment:
"module" is a new term - mentioned in one place above. What does it mean? Do we need it?